src/backend.ml

```diff
@@ -897,7 +879,7 @@ let back_tick = List.hd (Ulib.Text.explode (r"`"))
 let quotation_mark = List.hd (Ulib.Text.explode (r"\""))
 let backslash = List.hd (Ulib.Text.explode (r"\\"))
 
-module Isa : Target = struct
+module Isa () : Target = struct
```

Comment:
@bauereiss I had to functorize this module, because `typedefrec_start` depends on a reference that is set after static module initialization.